src/adcm_pytest_plugin/fixtures.py

```diff
@@ -141,6 +141,8 @@ def image(request, cmd_opts, adcm_api_credentials, additional_adcm_init_config):
 def _adcm(image, cmd_opts, request, adcm_api_credentials, upgradable=False) -> Generator[ADCM, None, None]:
     repo, tag = image
     labels = {"pytest_node_id": request.node.nodeid}
+    if hasattr(cmd_opts, "debug_owner") and cmd_opts.debug_owner:
+        labels["debug_owner"] = cmd_opts.debug_owner
```

Comment:
There is no information about this option in this plugin. Perhaps you should think about how to transfer the desired labels in a different way?

Reply:
This is a bit controversial question. I guess we can leave an extended comment here.

Reply:
Comment added